invoker/requirements.txt

```diff
@@ -0,0 +1,3 @@
+protobuf==3.4.0
```

Comment:
@dturanski Now that the generated code has been checked in, can this dependency be deleted?